tari-project/universe
# feat(ui): log in button and logged in user ui

Description
---

This PR adds a designed "Log in to claim gems" button and a Logged in user UI to the top right corner of the APP. 

Right now this is static UI, and is not fully integrated yet. Integration work will continue in this branch and wont be merged until complete. 

Motivation and Context
---

New Button:

![CleanShot 2024-08-30 at 15 09 17](https://github.com/user-attachments/assets/a0d68e1e-1e7b-473b-88d2-8b39c6366537)

New Logged In User UI:

![CleanShot 2024-08-30 at 15 10 19](https://github.com/user-attachments/assets/c34d0704-ec5b-4ceb-a263-588bf5ca30f5)

New User Menu with Logout Button

![CleanShot 2024-08-30 at 15 10 46](https://github.com/user-attachments/assets/0ef540a3-27c1-47ba-b5ee-f60b1f556b01)

How Has This Been Tested?
---

Locally, Mac OS M1 Pro

What process can a PR reviewer use to test or verify this change?
---

1. Click the "Log in to claim gems" button and get redirected to the Airdrop site. 
2. Log in with your X account using the modal on the Airdrop site. Complete the login process.
3. Once complete on the Airdrop site, Go back to the APP. You should see the logged in user info in the top corner instead of the button.
4. Click the your Avatar in the top right to open the menu, and click "Logout". The User info should vanish and the Log in button should reappear. 

Breaking Changes
---
